.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) HashiCorp, Inc.
```

Comment:
FYI: I asked whether issue templates should be included or not, and we're still waiting for a response from Legal but I'm guessing the answer will be these don't need the header: https://hashicorp.slack.com/archives/C03CR2GBBB2/p1675361917510739